src/backend/base/langflow/components/models/AIMLModel.py

```diff
@@ -1,148 +1,128 @@
-            value=0.2,
-            info="Controls the creativity of model responses.",
+        SecretStrInput(
+            name="api_key",
+            display_name="AIML API Key",
+            info="The AIML API Key to use for the OpenAI model.",
+            advanced=False,
+            value="AIML_API_KEY",
         ),
-        StrInput(
-            name="system_message",
-            display_name="System Message",
-            info="System message to pass to the model.",
+        FloatInput(name="temperature", display_name="Temperature", value=0.1),
+        IntInput(
+            name="seed",
+            display_name="Seed",
+            info="The seed controls the reproducibility of the job.",
             advanced=True,
+            value=1,
         ),
     ]
 
-    def make_request(self) -> Message:
-        api_key = SecretStr(self.aiml_api_key) if self.aiml_api_key else None
-
-        headers = {
-            "Content-Type": "application/json",
-            "Authorization": f"Bearer {api_key.get_secret_value()}" if api_key else "",
-        }
-
-        messages = []
-        if self.system_message:
-            messages.append({"role": "system", "content": self.system_message})
+    def build_model(self) -> LanguageModel:  # type: ignore[type-var]
+        output_schema_dict: dict[str, str] = reduce(operator.ior, self.output_schema or {}, {})
+        aiml_api_key = self.api_key
+        temperature = self.temperature
+        model_name: str = self.model_name
+        max_tokens = self.max_tokens
+        model_kwargs = self.model_kwargs or {}
+        aiml_api_base = self.aiml_api_base or "https://api.aimlapi.com"
+        json_mode = bool(output_schema_dict) or self.json_mode
+        seed = self.seed
 
-        if self.input_value:
-            if isinstance(self.input_value, Message):
-                # Though we aren't using langchain here, the helper method is useful
-                message = self.input_value.to_lc_message()
-                if message.type == "human":
-                    messages.append({"role": "user", "content": message.content})
-                else:
-                    raise ValueError(f"Expected user message, saw: {message.type}")
-            else:
-                raise TypeError(f"Expected Message type, saw: {type(self.input_value)}")
+        if isinstance(aiml_api_key, SecretStr):
+            openai_api_key = aiml_api_key.get_secret_value()
         else:
-            raise ValueError("Please provide an input value")
+            openai_api_key = aiml_api_key
 
-        payload = {
-            "model": self.model_name,
-            "messages": messages,
-            "max_tokens": self.max_tokens or None,
-            "temperature": self.temperature or 0.2,
-            "top_k": self.top_k or 40,
-            "top_p": self.top_p or 0.9,
-            "repeat_penalty": self.repeat_penalty or 1.1,
-            "stop_tokens": self.stop_tokens or None,
-        }
+        model = ChatOpenAI(
```

Comment:
Was curious about this usage - does using `OpenAI`'s implementation have any compatibility issues using other models?